vmware_aria_operations_integration_sdk/docker_wrapper.py

```diff
@@ -185,11 +186,23 @@ def build_image(
             **kwargs,
         )
     except docker.errors.BuildError as error:
+        error_message = f"{error}"
+        for log in error.build_log:
+            if "stream" in log:
+                line = remove_ansi_escape_sequences(log["stream"])
```

Comment:
Can you document what escape characters are present in the stream and why we need to remove them?